trulens_eval/trulens_eval/feedback/groundedness.py

```diff
@@ -113,16 +113,18 @@ def groundedness_measure_with_cot_reasons(
                 "Only LLM providers are supported for groundedness_measure_with_cot_reasons."
             )
         else:
-            reason = self.groundedness_provider._groundedness_doc_in_out(
-                source, statement
-            )
-            i = 0
-            for line in reason.split('\n'):
-                if "Score" in line:
-                    groundedness_scores[f"statement_{i}"
-                                       ] = re_0_10_rating(line) / 10
-                    i += 1
-        return groundedness_scores, {"reasons": reason}
+            hypotheses = sent_tokenize(statement)
+            reasons_str = ""
+            for i, hypothesis in enumerate(tqdm(
```

Comment:
tqdm is here for debugging? If so, remove?

Reply:
tqdm is here to show the progress bar for feedback generation - we have this for the nli method already.

Reply:
Better to not have it for either? I don't have a strong opinion